### DISCUSSION TITLE
Default configuration values: best, skip_if_unavailable

### DISCUSSION THREAD

**Discussion by j-mracek** [rpm-software-management/dnf5#564] [Q&A]
Configuration option best and skip_if_unavailable are important to identify broken packages in distribution or unavailable repositories. When those options use strict values they allow to detect potential problems in distribution or identify unavailable repository with critical patches. When options uses relax setting it creates user more friendly environment because all scripts pass because problems are skipped, patches are not applied. In the last few years Fedora project moved into more stable state therefore we can think about new defaults.

DNF uses various default and default overrides for RHEL and Fedora

In libdnf project we have
skip_if_unavailable=True=false
best=False

Fedora downstream dnf.conf
skip_if_unavailable=True=True
best=False

RHEL downstream dnf.conf
skip_if_unavailable=True=False
best=True

I thing it is good time to move our security level to higher level therefore we can use strict behavior for all distributions:
In dnf5 project we could use 
skip_if_unavailable=True=false
best=true

With no downstream overrides. `/etc/dnf/dnf.conf` will be reserved for user specific overrides and not for downstream overrides.

**Comment by kontura** [comment]
I like the proposal of:
```
skip_if_unavailable=false
best=true
```
in dnf5 upstream with no downstream overrides.
Like mentioned these are more secure and I think better defaults.

Also it would be nice to have the same config everywhere.
